controller/controller.go

```diff
@@ -214,20 +214,25 @@ func (c *Controller) processCluster(updateCtx context.Context, workerNum uint, c
 	cluster := clusterInfo.Cluster
 	clusterLog := c.logger.WithField("cluster", cluster.Alias).WithField("worker", workerNum)
 
-	clusterLog.Infof("Processing cluster (%s)", cluster.LifecycleStatus)
+	versionedLog := clusterLog
+	if clusterInfo.NextVersion != nil {
+		versionedLog = clusterLog.WithField("version", clusterInfo.NextVersion.String())
+	}
+
+	versionedLog.Infof("Processing cluster (%s)", cluster.LifecycleStatus)
 
 	err := c.doProcessCluster(updateCtx, clusterLog, clusterInfo)
```

Comment:
doProcessCluster uses clusterLog and therefore does not log cluster version to avoid clutter.